app/jobs/subscription_confirm_job.rb

```diff
@@ -45,27 +45,33 @@ def recently_closed_order_cycles
   def confirm_order!(order)
     record_order(order)
 
-    if process_payment!(order)
-      send_confirmation_email(order)
-    else
+    process_payment!(order)
+    send_confirmation_email(order)
+  rescue StandardError => e
```

Comment:
This will now handle errors like https://app.bugsnag.com/yaycode/openfoodnetwork-uk/errors/5f104f65e9f6490017ad5250?filters[event.since][0][type]=eq&filters[event.since][0][value]=30d&filters[error.status][0][type]=eq&filters[error.status][0][value]=open&filters[search][0][type]=eq&filters[search][0][value]=delayed, which just happened.